src/components/ChannelList/hooks/listeners/useChannelTruncated.js

```diff
@@ -0,0 +1,23 @@
+import { useContext, useEffect } from 'react';
+
+import { ChatContext } from '../../../../context';
+
+export const useChannelTruncated = ({
+  onChannelTruncated,
+  setChannels,
+  setForceUpdate,
+}) => {
+  const { client } = useContext(ChatContext);
+
+  useEffect(() => {
+    const handleEvent = (e) => {
+      if (onChannelTruncated && typeof onChannelTruncated === 'function') {
```

Comment:
made this comment in few places
```suggestion
      if (typeof onChannelTruncated === 'function') {
```